src/js/client/common-jsondev-widget.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Common JSON Dev Widget
+ *
+ * Contains a grade `gpii.devpmt.jsonDev` that can be used in
+ * various places on the page to display underlying JSON for
+ * development and debugging purposes.
+ *
+ * Copyright 2019 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.devpmt");
+fluid.registerNamespace("gpii.devpmt.jsonDev");
```

Comment:
Only the last of these 3 is necessary

Reply:
removed

Reply:
Please tidy up similar multiple registrations in all the other files